tests/unit/tasks/datasets/test_push.py

```diff
@@ -43,37 +43,35 @@ def test_push_to_huggingface(
 
         result = cli_runner.invoke(
             cli,
-            "datasets push-to-huggingface --name my-dataset --workspace my-workspace --repo-id argilla/my-dataset --private",
+            "datasets --name my-dataset --workspace my-workspace push-to-huggingface --repo-id argilla/my-dataset --private",
         )
 
         assert result.exit_code == 0
         dataset_from_argilla_mock.assert_called_once_with(name="my-dataset", workspace="my-workspace")
         push_to_huggingface_mock.assert_called_once_with(repo_id="argilla/my-dataset", private=True, token=None)
 
-    @pytest.mark.parametrize(
-        "args, exception_msg",
-        [
-            ("--repo-id argilla/my-dataset", "Missing option"),
-            ("--name my-dataset", "Missing option"),
-        ],
-    )
-    def test_push_to_huggingface_missing_args(
+    def test_push_to_huggingface_missing_repo_id_arg(
         self,
         cli_runner: "CliRunner",
         cli: "Typer",
-        args: str,
-        exception_msg: str,
+        mocker: "MockerFixture",
+        remote_feedback_dataset: "RemoteFeedbackDataset",
     ) -> None:
-        result = cli_runner.invoke(cli, f"datasets push-to-huggingface {args}")
+        mocker.patch(
+            "argilla.client.feedback.dataset.local.FeedbackDataset.from_argilla",
+            return_value=remote_feedback_dataset,
+        )
+
+        result = cli_runner.invoke(cli, "datasets --name my-dataset push-to-huggingface")
```

Comment:
With the current approach for the CLI as `datasets [NAME] [WORKSPACE] [CMD] [ARGS]` shouldn't we make `CMD` show the `--help` by default? Or do we want it to fail if no args are provided? I guess it would depend on whether we have commands without args or not, right?

Reply:
I mean whether we should include `no_args_is_help=True` the the CMD e.g. `push-to-huggingface`

Reply:
ah, I see! I think it's better to not show by default the `--help`